### ISSUE TITLE
Custom IAM policy doesn't work with private subnets

### ISSUE THREAD

**Issue by mattmanning** [convox/rack#1180]
I *think* the private subnets are the issue here. Seems to be the main difference from a standard install.

```
$ convox install \
>  --private \
>  --instance-count 3 \
>  --instance-type t2.micro \
>  --region us-west-2 \
>  --vpc-cidr 10.254.0.0/16 \
>  --subnet-cidrs "10.254.100.0/24,10.254.101.0/24,10.254.102.0/24" \
>  --private-cidrs "10.254.200.0/24,10.254.201.0/24,10.254.202.0/24" \
>  ./Downloads/credentials.csv 


     ___    ___     ___   __  __    ___   __  _
    / ___\ / __ \ /  _  \/\ \/\ \  / __ \/\ \/ \
   /\ \__//\ \_\ \/\ \/\ \ \ \_/ |/\ \_\ \/>  </
   \ \____\ \____/\ \_\ \_\ \___/ \ \____//\_/\_\
    \/____/\/___/  \/_/\/_/\/__/   \/___/ \//\/_/


Installing Convox (20160901022059)...
(Private Network Edition)
Email Address (optional, to receive project updates): 
Created CloudWatch Log Group: convox-d1hv01-LogGroup-19FV480LWBKP7
Failed AWS::DynamoDB::Table: Resource creation cancelled
Failed AWS::S3::Bucket: Resource creation cancelled
Failed AWS::DynamoDB::Table: Resource creation cancelled
Failed AWS::IAM::Role: Resource creation cancelled
Failed AWS::S3::Bucket: Resource creation cancelled
Failed AWS::IAM::Role: Resource creation cancelled
Failed AWS::EC2::EIP: API: ec2:allocateAddress You are not authorized to perform this operation.
Failed AWS::IAM::User: Resource creation cancelled
Failed AWS::IAM::User: Resource creation cancelled
Failed AWS::IAM::Role: Resource creation cancelled
Failed AWS::EC2::VPC: Resource creation cancelled
Failed AWS::SNS::Topic: Resource creation cancelled
Failed AWS::IAM::Role: Resource creation cancelled
Failed AWS::EFS::FileSystem: Resource creation cancelled
Failed AWS::EC2::InternetGateway: Resource creation cancelled
Failed AWS::EC2::EIP: API: ec2:allocateAddress You are not authorized to perform this operation.
Failed AWS::EC2::EIP: API: ec2:allocateAddress You are not authorized to perform this operation.
Created ECS Cluster: convox-d1hv01-Cluster-1FG8GQECZ1HS8
Deleted ECS Cluster: convox-d1hv01-Cluster-1FG8GQECZ1HS8
Deleted CloudWatch Log Group: convox-d1hv01-LogGroup-19FV480LWBKP7
Deleted NAT Elastic IP: 
Deleted NAT Elastic IP: 
Skipped S3 Bucket: convox-d1hv01-settings-1tqzcye38kpm8
Skipped S3 Bucket: convox-d1hv01-registrybucket-dgz8sjmzxsep
Deleted NAT Elastic IP: 
Deleted IAM User: convox-d1hv01-KernelUser-1VDE4IHUNHGR1
Deleted IAM User: convox-d1hv01-RegistryUser-AK6GTYS4RWYG
Deleted VPC Internet Gateway: igw-5039e634
Deleted VPC: vpc-f4320c90
Deleted DynamoDB Table: convox-d1hv01-releases
Deleted DynamoDB Table: convox-d1hv01-builds
Deleted EFS Filesystem: fs-d802ff71
ERROR: stack creation failed, contact support@convox.com for assistance
```

**Comment by mattmanning** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;tLpILWAn&#x2F;645-custom-iam-policy-doesn-t-work-with-private-subnets"><img src="https:&#x2F;&#x2F;github.trello.services&#x2F;images&#x2F;trello-icon.png" width="12" height="12"> Custom IAM policy doesn&#39;t work with private subnets</a>

**Comment by MiguelMoll** [comment]
Fixed via convox/site#253
